DigitalExcellence/dex-backend
# created a user self endpoint

## Description

This endpoint adds a new endpoint for the user to get their own information, this is usefull for the frontend to get the users privileges etc.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. login to the identity server
2. make a request to the api endpoint /api/user 
3. you should get your current information

## Link to issue

Closes: #141
